appveyor_install.ps1

```diff
@@ -66,6 +66,34 @@ Start-Process python -ArgumentList "-m pip install psutil pyYaml six ccm" -Wait
 Write-Host "Set execution Policy"
 Set-ExecutionPolicy -ExecutionPolicy Unrestricted -Scope Process
 
+
+#removing any existing ccm cluster
+Write-Host "Removing any existing ccm clusters"
+
+$params = "/c ccm list"
+& "cmd.exe" $params | Tee-Object -Variable scriptOutput | Out-Null
+
+If ($scriptOutput)
+{
+  $list = $scriptOutput.Split(" ")
+  Write-Host "[ccm] list $($list)"
+
+  Foreach ($cluster in $list)
+  {
+    If (-Not $cluster.equals(""))
+    {
+      $name = $cluster.Replace("*", "")
+      & "cmd.exe" "/c ccm switch $($name)" | Tee-Object -Variable result | Out-Null
```

Comment:
I think this is not needed, you can remove a ccm cluster by name without switching:
```shell
ccm remove <cluster_name>
```